README.md

```diff
@@ -1,6 +1,6 @@
 # Datadog Agent release for BOSH
 
-* For debian and rpm based stemcells (more to come)
+* For debian and rpm based stemcells
```

Comment:
nit: Red Hat (or RHEL), not RPM.

Or even better, RHEL/CentOS?

nit 2: s/debian/Debian.